src/sonic-yang-models/yang-models/sonic-default-lossless-buffer-parameter.yang

```diff
@@ -0,0 +1,53 @@
+module sonic-default-lossless-buffer-parameter {
+
+    yang-version 1.1;
+
+    namespace  "http://github.com/Azure/sonic-default-lossless-buffer-parameter";
+
+    prefix default-lossless-buffer-parameter;
+
+    description "DEFAULT_LOSSLESS_BUFFER_PARAMETER YANG module for SONiC OS";
+
+    revision 2022-05-31 {
+        description "Initial version";
+    }
+
+    container sonic-default-lossless-buffer-parameter {
+        container DEFAULT_LOSSLESS_BUFFER_PARAMETER {
+
+            description "DEFAULT_LOSSLESS_BUFFER_PARAMETER part of config_db.json";
+
+            list DEFAULT_LOSSLESS_BUFFER_PARAMETER_LIST {
+                key "name";
+
+                leaf name {
+                    type string {
+                        pattern '[a-zA-Z0-9]{1}([-a-zA-Z0-9_]{0,31})' {
+                            error-message "Invalid default lossless buffer parameter list name.";
+                            error-app-tag default-lossless-buffer-parameter-invalid-list-name;
+                        }
+                        length 1..32 {
+                            error-message "Invalid length for the default lossless buffer parameter list name.";
+                            error-app-tag default-lossless-buffer-parameter-invalid-list-name-length;
+                        }
+                    }
+                }
+
+                leaf default_dynamic_th {
+                    type int32 {
```

Comment:
Can this be int8?

Reply:
what's the benefit of using int8 and what's the common practice in the community?

Reply:
It is better to align with the smallest possible type that can accomodate the range.

Reply:
As discussed offline, it's a common practice to use the smallest possible type.
Fixed.